packit_service/worker/handlers/fedmsg_handlers.py

```diff
@@ -124,21 +124,19 @@ def run(self) -> HandlerResults:
 @add_topic
 @add_to_mapping
 class NewDistGitCommitHandler(FedmsgHandler):
-    """ A new flag was added to a dist-git pull request """
+    """ ??? A new flag was added to a dist-git pull request """
```

Comment:
?

Reply:
NewDistGitCommit != new flag was aded to dg pull-request

Reply:
so just update the docstring then, it was probably just copy-pasted

Reply:
I was not sure and (main reason) had no time to check the code properly to fix it. I will correct that.